torch/csrc/cuda/shared/cudart.cpp

```diff
@@ -93,10 +99,17 @@ void initCudartBindings(PyObject* module) {
         return C10_CUDA_ERROR_HANDLED(cudaStreamDestroy((cudaStream_t)ptr));
       });
 #if !defined(USE_ROCM)
+#if CUDA_VERSION < 12000
   cudart.def(
       "cuda"
       "ProfilerInitialize",
       cudaProfilerInitialize);
+#else
+  cudart.def(
+      "cuda"
+      "ProfilerInitialize",
+      cudaProfilerStart);
```

Comment:
I'm not sure if this is correct. `cudaProfilerStart` and `cudaProfilerInitialize` appear different functions, and there is already a python function for `cudaProfilerStart`:

https://github.com/pytorch/pytorch/blob/95a115dd07564f32e7ddca77c7f03fe2badd71dd/torch/csrc/cuda/shared/cudart.cpp#L48-L60

Also, per this NVIDIA person, calling `cudaProfilerInitialize` may not be needed anymore.
 https://forums.developer.nvidia.com/t/cudaprofilerinitialize-is-deprecated-alternative/200776/4